build_functions/CMakeLists.txt

```diff
@@ -193,3 +193,22 @@ function(compileTargetAsC11 theTarget)
     set_target_properties(${theTarget} PROPERTIES CXX_STANDARD 11)
   endif()
 endfunction()
+
+set(LIBS_TO_BE_REAL_CHECKED CACHE INTERNAL "LIBS_TO_BE_CHECKED")
+
+function (add_library())
+    _add_library(${ARGV})
+    get_target_property(lib_type ${ARGV0} TYPE)
+    if( (NOT ${CMAKE_CURRENT_LIST_DIR} MATCHES "deps") AND # ignore dependency libs
```

Comment:
>"deps" [](http://example.com/codeflow?start=47&length=6)

So if I have a lib named deps_blah.lib this will trigger. Can we make the matching better please?

Reply:
done in next iteration.